### ISSUE TITLE
Filter By ingredient suggestion list in ItemsRepeater is not announced.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#578]
**Describe the bug**

**Steps to reproduce the bug**

1. Turn on Narrator
2. Launch XAML Controls Gallery.
3. Activate ItemsRepeater from home page, navigate to header "Virtualized, Content-Heavy layout with filtering and Sorting" and type any any recipe e.g. "Watermelon" in Filter By ingredient textbox and observe.

**Expected behavior**

Narrator should announce the suggestion lists available.

**Actual behavior**

Narrator remains silent and does not inform the users about the available suggestions on typing any recipe.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97391742-27b46d00-189d-11eb-9c18-7e6c99971299.png)

**Version Info**

Gallery version (found on Settings page): 1.2.13.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/25440948/)

**Comment by chingucoding** [comment]
Which part of the sample would be the correct element to raise the property changed here? Should the repeater raise a propertychanged event?

**Comment by YuliKl** [comment]
> Which part of the sample would be the correct element to raise the property changed here? Should the repeater raise a propertychanged event?

@anawishnoff, @ranjeshj any suggestions?

**Comment by anawishnoff** [comment]
Is the issue that the narrator doesn't read out what's on the recipe card? Or should the narrator read out the number of results available before filtering? That would probably make more sense to me, and I think in that case we'd want the repeater to raise a propertychanged event or some kind of other event that shows the number of visible items has changed?

**Comment by chingucoding** [comment]
The issue is that typing in the textbox does not allow users with narrator to notice that the collection of recipes has changed, typing in that textbox only announces the typed character- So users using narrator have no way of knowing the connectio between the textbox and the collection of recipes. I think announcing the number of items makes the most sense here, but maybe there is a better way to solve this?

**Comment by anawishnoff** [comment]
Ah, I see! I think a good first step (an action item on me) would be to add some textual context underneath the title of the sample to explain what it does ("as you type, the list of recipes will become filtered with your keyword"). Then, I think it would be useful to announce how many items are in currently in the repeater (maybe "25 out of 50 items showing"). 

But, I feel like this could get really annoying and overwhelming if it happened after every keystroke. Is it possible to monitor the time since last keystroke and play the narration on, say, 3 seconds after typing has stopped? I might be reaching here, just checking if that's possible.

**Comment by chingucoding** [comment]
Announcing the change everytime a key was entered can definitely be overwhelming. I'm not sure if adding a delay is the right choice though. Usually, property changed events get raised immediately, not after a delay. But I am not sure if there are any guidelines for this kind of scenario. @YuliKl do you know what the best practice would be in that case?
